examples/using-reach-skip-nav/README.md

```diff
@@ -0,0 +1,32 @@
+# Using @reach/skip-nav
+
+Demonstrates the usage of [@reach/skip-nav](https://reacttraining.com/reach-ui/skip-nav/)
+
+## Add SkipNavLink and SkipNavContent to your Layout
+
+```javascript:layout.js
+import { SkipNavLink, SkipNavContent } from "@reach/skip-nav"
+import "@reach/skip-nav/styles.css" //this will show/hide the link on focus
+;<>
```

Comment:
I wonder if for clarity for users, it would be easier to see this wrapped in a function called App?